keyboards/syenakeyboards/aswagata/info.json

```diff
@@ -0,0 +1,41 @@
+{
+    "manufacturer": "Syenasweta",
+    "keyboard_name": "SyenaKeyboards (Aswagata)",
+    "maintainer": "syenasweta",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": true,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "lto": true,
+        "key_override": true
```

Comment:
* LTO is not enabled like this — belongs under a different key. Not necessary for RP2040 anyway.
* Key overrides cannot be enabled at the keyboard level.
```suggestion
        "nkro": true
```